ios/Podfile.lock

```diff
@@ -205,7 +205,7 @@ PODS:
     - React
   - react-native-splash-screen (3.2.0):
     - React
-  - react-native-webview (8.0.6):
+  - react-native-webview (8.0.7):
```

Comment:
You need to run `make nix-update-pods`.

Reply:
@jakubgs ran it, but no new files to commit

Reply:
Oh wait, my bad, it's fine as it is.